rsconnect/bundle.py

```diff
@@ -1418,8 +1422,13 @@ def get_default_entrypoint(directory):
     if len(python_files) == 1:
         return python_files[0][:-3]
 
-    logger.warning("Can't determine entrypoint; defaulting to 'app'")
-    return "app"
+    # try app-*.py, app_*.py, *-app.py, *_app.py
+    app_files = list(filter(lambda s: re_app_prefix.match(s) or re_app_suffix.match(s), python_files))
+    if len(app_files) == 1:
+        # In these cases, the app should be in the "app" attribute
+        return app_files[0][:-3] + ":app"
```

Comment:
If you omit the attribute name, Connect will use the Flask import rules to look for common attribute names in the imported module (`app`, `application`, or the `create_app` or `make_app` factory functions).

Connect will also strip the `.py` suffix from the entrypoint module.

Reply:
OK, thanks! It looks like this docstring might be wrong then? If so, I can update it. https://github.com/rstudio/rsconnect-python/blob/f4def0c590fef96f6ee8aa8ea31512629b94a50c/rsconnect/bundle.py#L1428C60-L1429C74

Reply:
Yeah, that docstring looks wrong.